packages/forgae-lib/contractTypes.ts

```diff
@@ -0,0 +1,114 @@
+export type KeyPair = {
+    publicKey: string,
+    secretKey: string
+}
+
+
+export type ParsedContractFunction = {
+    name?: string,
+    args?: Array<string>,
```

Comment:
arguments must also always be passed, even though sometimes is just an empty array

Reply:
Still not fixed

Reply:
We are getting compile time error "Property 'args' is missing in type 'AciFunctions' but required in type 'ParsedContractFunction", which is due to inconsistency between properties of AciFunctions & ParsedContractFunction - one is args & and the other is arguments. This is coming outside the deployer and will be standardized later on.